source/extensions/tracers/datadog/agent_http_client.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <datadog/http_client.h>
+
+#include <string>
+
+#include "envoy/http/async_client.h"
+
+#include "source/common/common/logger.h"
+#include "source/common/upstream/cluster_update_tracker.h"
+#include "source/extensions/tracers/datadog/dd.h"
+
+#include "absl/container/flat_hash_map.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace Datadog {
+
+struct TracerStats;
+
+class AgentHTTPClient : public dd::HTTPClient,
+                        public Http::AsyncClient::Callbacks,
+                        private Logger::Loggable<Logger::Id::tracing> {
+public:
+  struct Handlers {
+    ResponseHandler on_response;
+    ErrorHandler on_error;
+  };
+
+private:
```

Comment:
we typically put the private member vars at the end of the class definition

https://google.github.io/styleguide/cppguide.html#Class_Format

Reply:
Ok, I moved the private members of all classes in header files.